### ISSUE TITLE
Skeleton2D IK targets the wrong node when running in multiplayer

### ISSUE THREAD

**Issue by chrisl8** [godotengine/godot#86945]
### Tested versions

v4.2.1.stable.official [b09f793f5]

### System information

Godot v4.2.1.stable - Windows 10.0.22621 - GLES3 (Compatibility) - NVIDIA GeForce RTX 3080 Ti (NVIDIA; 31.0.15.4633) - AMD Ryzen 9 5900X 12-Core Processor (24 Threads)

### Issue description

I expect when setting up a Skeleton2D with a Modification Stack with a Target Nodepath that each instanciated instance of that Skeleton will follow the Target Nodepath that is local to that scene.

However, instead EVERY instance of the spawned skeleton follows the SAME Target Nodepath.

We worked on this for about 48 hours but were unable to find any way around this.

### Steps to reproduce

You must set up your Debug->Run Multiple Instances to at least 3.

One of the windows opened will be the server, and will basically look blank, just ignore it.

The other two will focus on yoru characters.

You can use WASD to move the character around if you need to prove which window you are controlling and/or which character is native to that window.

Move the mouse around in front of both windows and you will see that BOTH IK skeletons follow the SAME pointer.
They are both following ONE of the two (or 3) Player/Left Hand Target nodes, rather than each following the Player/Left Hand Target node within their respective player.

https://github.com/godotengine/godot/assets/6188278/b7a03272-d56f-42b9-8426-34b9266435b4

### Minimal reproduction project (MRP)

https://github.com/chrisl8/godot-ik-broken-multiplayer

or

[godot-ik-broken-multiplayer.zip](https://github.com/godotengine/godot/files/13855347/godot-ik-broken-multiplayer.zip)

**Comment by chrisl8** [comment]
Reopening the project should make it open. Not sure why Godot claims it is corrupt though.

Does setting it to local actually make both players work or just disable one of them entirely?

**Comment by kleonc** [comment]
> Reopening the project should make it open. Not sure why Godot claims it is corrupt though.

Reopening indeed made the error go away. :upside_down_face:

> Does setting it to local actually make both players work or just disable one of them entirely?

Oh, it rather disables one. But enabling `resource_local_to_scene` for both SkeletonModificationStack2D and SkeletonModification2DTwoBoneIK does the thing for me, works even for 4 windows / 3 players.
![Godot_v4 2 1-stable_win64_OZCvds6gCd](https://github.com/godotengine/godot/assets/9283098/71802e0a-c4d0-450b-8a6c-75731db87f0f)

**Comment by chrisl8** [comment]
Thank you again so much for the further explanation!

And sorry for confusing your reply by editing mine.

**Comment by chrisl8** [comment]
@kleonc  Thank you so much again for taking time to respond to my issue report and for explaining the solution so clearly. Indeed I had found, actually days ago, the "Local to Scene" checkbox, **but I did not find the second one**, so your screenshot was the final solution for my complete understanding.

I also found the script in [this comment](https://github.com/godotengine/godot/issues/73888#issuecomment-1768236299) to be ideal, as it was easy to add that to each Skeleton in the project and then my other team mate doesn't have to worry about checking the right boxes.

I will close this as a duplicate of Issue #73888 and will subscribe to that issue to watch for solutions to test while using the workaround that you and others provided.
